README.md

```diff
@@ -90,3 +90,9 @@ If you don't have one, you can make the following changes to make GreeyBear spin
 
 1) In ```docker/env_file```, set the variable ```ELASTIC_ENDPOINT``` to ```http://elasticsearch:9200```.
 2) Add ```:docker/elasticsearch.yml``` to the last defined ```COMPOSE_FILE``` variable or uncomment the ```# local development with elasticsearch container``` block in ```.env``` file.
+
+
+
+## Documentation [![Documentation Status](https://readthedocs.org/projects/greedybear-docs/badge/?version=latest)](https://greedybear-docs.readthedocs.io/en/latest/?badge=latest)
```

Comment:
considering that Honeynet will manage the RTD instances, please move the link to https://greedybear.readthedocs.io/en/latest/

so change `greedybear-docs` with `greedybear` in both links

Reply:
I have this doubt , right from the start that how can I transfer my ownership over the instance of rtd docs once when I m done. Now it got cleared 😅